core/templates/dev/head/css/oppia.css

```diff
@@ -3890,6 +3890,129 @@ md-card.preview-conversation-skin-supplemental-card {
   }
 }
 
+.aggregated-stats-supplemental {
```

Comment:
This entire block should belong inside the respective html files, like this one: https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/library-page/library-page.directive.html

Notice that the styling is encapsulated inside style block